pkgs/top-level/all-packages.nix

```diff
@@ -8516,6 +8516,8 @@ in
 
   qt4 = self.kde4.qt4;
 
+  qt4_clang = qt4.override { stdenv = clangStdenv; };
```

Comment:
One more moment -- please wrap this into `lowPrio` so that `nix-env` won't try to install this version. Like this:
`foo = lowPrio (foo.override ...);`